app/components/export_with_count_button/view.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<% if count.zero? %>
+  <%= govuk_inset_text do %>
+    <%= no_record_text %>
+  <% end %>
+<% end %>
+
+<%= govuk_link_to "#{button_text} (#{count} #{count_label.pluralize(count)})", href, class: "govuk-button" %>
```

Comment:
sorry for stupid question but why do we show the button if there are no records?

Reply:
Yeah I don't know the designs showed both. I guess maybe from a ux point of view to always allow the action rather than hide/prevent it? @andrewscrivener could confirm